pages/about/index.tsx

```diff
@@ -178,165 +182,210 @@ const ArticlesList = ({ articles }: ArticlesListProps): JSX.Element => {
-          />
-        </a>
-
-        <h3 className="mb-1 sm:my-4">{t('about_page.our_team_autumn_title')}</h3>
-        <a href="https://data.movapp.eu/images/team/large-team-photo-autumn.jpg" target="_blank" rel="noopener noreferrer">
-          <Image
-            src="https://data.movapp.eu/images/team/small-team-photo-autumn.jpg"
-            width="320"
-            height="180"
-            alt={t('about_page.our_team_current_title')}
-            className="hover:shadow-lg hover:-translate-y-1 hover:scale-110 hover:bg-indigo-500 duration-300 cursor-zoom-in"
-          />
-        </a>
-
-        <h3 className="mb-1 sm:my-4">{t('about_page.our_team_spring_title')}</h3>
-
-        <a href="https://data.movapp.eu/images/team/large-team-photo.jpg" target="_blank" rel="noopener noreferrer">
-          <Image
-            src="https://data.movapp.eu/images/team/small-team-photo.jpg"
-            width="320"
-            height="180"
-            alt={t('about_page.our_team_spring_title')}
-            className="hover:shadow-lg hover:-translate-y-1 hover:scale-110 hover:bg-indigo-500 duration-300 cursor-zoom-in"
-          />
-        </a>
-
-        {teams.map(({ team, members }) => (
-          <React.Fragment key={team[currentLanguage]}>
-            <H2>{team[currentLanguage]}</H2>
-            <P className="inline-block">{members}</P>
-          </React.Fragment>
-        ))}
+            />
+          </div>
+        </section>
+
+        <section className="md:mt-[124px] mt-20 px-4">
+          <div className="text-center mb-8">
+            <H2>{t('about_page.our_team_title')}</H2>
+          </div>
+
+          <SliderImageTeam />
+        </section>
+        <section className="md:grid md:grid-cols-3 md:gap-14 mb-16 md:mt-20 px-4 md:text-sm font-normal">
+          {teams.map(({ team, members }) => (
+            <React.Fragment key={team[currentLanguage]}>
+              <div>
+                <H2>{team[currentLanguage]}</H2>
+                <P className="inline-block">{members}</P>
+              </div>
+            </React.Fragment>
+          ))}
+        </section>
+
+        <section className="flex justify-center px-4">
+          <div className="my-4 place-content-around flex-wrap md:flex-nowrap xl:max-w-[900px] md:max-w-[600px] h-52 flex md:p-16  bg-white border-b-primary-grey rounded-3xl shadow-md align-middle items-center">
+            <Image src={help} alt="help" />
+
+            <div className="md:ml-16 xl:text-[30px] md:text-[24px] sm:text-[16px] p-4">
+              <Trans
+                i18nKey={'about_page.our_team_contact'}
+                components={[<TextLink href={`/contacts`} locale={currentLanguage} target="_self" key="contacts" />]}
+              />
+            </div>
+          </div>
+        </section>
+
+        <section className="max-w-[1185px] md:mt-[124px] mt-20 mx-auto p-4">
+          <div className="text-center mb-16 ">
```

Comment:
Ještě prosím zmenšit margin bottom na malé obrazovce. Týká se to i nadpisu said_about_us.